package.json

```
@@ -18,12 +18,19 @@
   },
   "devDependencies": {
     "lerna": "^3.22.1",
-    "@commitlint/cli": "^11.0.0",
-    "@commitlint/config-conventional": "^11.0.0",
-    "@commitlint/config-lerna-scopes": "^11.0.0",
-    "commitlint": "^11.0.0",
-    "husky": "^4.3.0",
-    "prettier-standard": "^16.4.1"
+    "prettier-standard": "^16.4.1",
+    "sass-loader": "^10.1.1",
+    "node-sass": "^5.0.0",
+    "babel-eslint": "^10.1.0",
+    "eslint": "^7.18.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-nuxt": "^2.0.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-vue": "^7.5.0",
+    "prettier": "^2.2.1",
+    "stylelint": "^13.9.0",
+    "stylelint-config-prettier": "^8.0.2",
+    "stylelint-config-standard": "^20.0.0"
```

These dependencies are common to all packages or useful to manage the repository. You are are adding dependencies only used in `ui` package and removing commitlint and husky ones. For that reason, `prettier-standard` can be removed from here because is only used in `api`

```suggestion
  "@commitlint/cli": "^11.0.0",
    "@commitlint/config-conventional": "^11.0.0",
    "@commitlint/config-lerna-scopes": "^11.0.0",
    "commitlint": "^11.0.0",
    "husky": "^4.3.0",
```